kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/java/org/kie/server/services/jbpm/ui/ImageServiceBase.java

```diff
@@ -92,27 +93,39 @@ public ImageServiceBase(RuntimeDataService dataService, Map<String, ImageReferen
     }
 
     public String getProcessImage(String containerId, String processId) {
+        return getProcessImage(containerId, processId, false);
+    }
+
+    public String getProcessImage(String containerId, String processId, boolean addSvgViewBox) {
         containerId = registry.getContainerId(containerId, ContainerLocatorProvider.get().getLocator());
 
         String imageSVGString = null;
         byte[] imageSVG = getProcessImageAsBytes(containerId, processId);
         if (imageSVG != null) {
-            try {
-                imageSVGString = new String(imageSVG, "UTF-8");
-            } catch (UnsupportedEncodingException e) {
-                logger.debug("UnsupportedEncodingException while building process image due to {}", e.getMessage());
+            if (addSvgViewBox) {
```

Comment:
can't we handle this uniformly? i.e. remove the else branch completely and rely on the addSvgViewBox variable being passed to processor.getSVG(boolean) in both cases? it would help maintain encapsulation ...